src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import os
+import torch
+from torch._dynamo.backends.common import fake_tensor_unsupported
+from torch._dynamo.backends.registry import register_backend
+from torch._inductor.compile_fx import compile_fx
+
+from openvino.runtime import Core, Type, PartialShape
+from openvino.frontend import FrontEndManager
+from openvino.frontend.pytorch.decoder import TorchScriptPythonDecoder
+
+log = logging.getLogger(__name__)
+
+
+@register_backend
+@fake_tensor_unsupported
+def openvino(subgraph, example_inputs):
+    return ts_openvino(subgraph, example_inputs)
+
+
+def ts_openvino(subgraph, example_inputs):
+    try:
+        model = torch.jit.script(subgraph)
```

Comment:
If we always have example_inputs, why cannot we just default to `torch.jit.trace` all the time? `torch.jit.script` is legitimate in the standalone application because user may not provide example input, but torchdynamo looks to be always using tracing as an essential part of the infrastructure. I can guess that we may have issues with dynamic shapes if always rely on tracing, but even now when we are using scripting, later in the code we always setting static shapes based on example_inputs so from this perspective we don't utilize any opportunities related to dynamic shapes.

Reply:
Yeah we can use jit.trace for now but we will have to switch it back once we enable dynamic shapes and control flow ops